tests/test_config.py

```diff
@@ -1,30 +1,27 @@
-from dynaconf import Dynaconf
 from nplinker.config import config
-from . import DATA_DIR
 
 
 def test_config_demo1():
     """Test loading the default config file (nplinker_demo1.toml)."""
-    # the file "nplinker_demo1.toml" is set in ./conftest.py
-    assert config.loglevel == "DEBUG"
-    assert config["loglevel"] == "DEBUG"
-    assert config.get("loglevel") == "DEBUG"
+    # The file "nplinker_demo1.toml" is set in ./conftest.py
 
-    assert config.logfile == ""
-    assert config.repro_file == ""
+    assert config.mode == "podp"
```

Comment:
Exactly, the magic happens in the file `config.py`.

`user_config_file = os.environ.get("NPLINKER_CONFIG_FILE", "nplinker.toml")`
- If user gives a toml file via env variable "NPLINKER_CONFIG_FILE", then it will be used. "nplinker_demo1.tom" is just this case, it's specified in the `tests/conftest.py` file.
- Otherwise, nplinker will search the `nplinker.toml` in the working directory (depending on where you execute nplinker). If found, then using it; otherwise, raise an error.